tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -55,6 +55,19 @@ If you are building Godot from source, make sure to follow the steps to enable
 .NET support in your build as outlined in the :ref:`doc_compiling_with_dotnet`
 page.
 
+Linux and \*BSD
+~~~~~~~~~~~~~~~
+
+On Linux and BSD systems you may have to link the .NET library path and "dotnet" executable. Example for Ubuntu 22.04 and Bash::
+
+    sudo ln -sT ~/.dotnet /usr/share/dotnet
+    sudo ln -sT ~/.dotnet/dotnet /usr/bin/dotnet
+
+Export .NET path in your shell:
+``~/.bashrc``::
+
+    export DOTNET_ROOT=$HOME/.dotnet
```

Comment:
Yes, I've used scripted install because I needed multiple versions of .NET for other projects. Packages available for Ubuntu sometimes are outdated, especially in case of exotic libraries. Scripted install is one of [the first options displayed prominently](https://learn.microsoft.com/en-ie/dotnet/core/install/linux?WT.mc_id=dotnet-35129-website).

> Since you are also only adding it to .bashrc this will only work if you launch Godot from a bash terminal and most users launch Godot from a GUI.

I'm launching *.AppImage with double click. Godot couldn't find the dotnet executable.

Reply:
> Scripted install is one of the first options displayed prominently.

The doc page you link already explains how to deal with paths when using scripted install. I'd expect whoever is using this installation method to follow that, instead of relying on our doc, right? Especially since it makes our doc less intelligible for everybody else (who installed using another method), and it will be kept up to date by the .NET team.